com.unity.mobile.notifications/Tests/Runtime/Android/AndroidNotificationSimpleTests.cs

```diff
@@ -3,11 +3,43 @@
 using UnityEngine;
 using UnityEngine.TestTools;
 using Unity.Notifications.Android;
+#if UNITY_EDITOR
+using UnityEditor;
+#endif
 
 class AndroidNotificationSimpleTests
+    : IPrebuildSetup, IPostBuildCleanup
 {
     const string kChannelId = "SerializeDeserializeNotificationChannel";
 
+#if UNITY_EDITOR
+    PluginImporter GetTestUtils()
+    {
+        var assets = AssetDatabase.FindAssets("UnityNotificationTestUtils");
+        if (assets.Length != 1)
+            throw new Exception("Something is wrong");
```

Comment:
Not a very good error, why not say 'Error while acquiring UnityNotificationTestUtils asset' or smth?